.ci/integrationTestDownstream.groovy

```diff
@@ -205,10 +209,7 @@ def runScript(Map params = [:]){
   unstash "source"
   dir("${BASE_DIR}"){
     withEnv(env){
-      sh """#!/bin/bash
-      export TMPDIR="${WORKSPACE}"
-      .ci/scripts/${agentType}.sh
-      """
+      sh(label: "Testing ${agentType}", script: ".ci/scripts/${agentType}.sh")
```

Comment:
Why adding the `.ci/scripts` to PATH if we use the relative path? Is there any other use case?

Reply:
to call `elasticdump` from python we use the command directly, and it is OK you can use a local instance of elasticdump when you run the test locally

Reply:
Cool! I did not see the connection with `elasticdump` from this changeset.

Thanks!